packages/core/src/components/button/button.tsx

```diff
@@ -74,9 +74,10 @@ export class AtomButton {
             [`atom-button`]: true,
             [`is-loading`]: this.loading,
             [`is-circle`]: this.shape === 'circle',
+            [`is-outline-filled`]: this.fill === 'outline-filled',
           }}
           color={this.disabled && !this.loading ? 'medium' : this.color}
-          fill={this.fill}
+          fill={this.fill === 'outline-filled' ? 'default' : this.fill}
```

Comment:
I didn't understand this validation, man. Why to we need this?

Reply:
its for typecheck, the type 'outline-filled' doesnt exists on ion-button, so it throws an error saying that this property cant be bind to this component, doing this, we prevent it

Reply:
hmm got it!